src/Commands/ProjectCreateCommand.php

```diff
@@ -468,8 +468,9 @@ function () use ($siteDir) {
                                 '# Put ignore patterns for css build artifacts and similar items below.',
```

Comment:
The comment needs to change too -- artifacts go above the line, so they are ignored in GitHub et. al., and committed on Pantheon.